python-checks/src/main/java/org/sonar/python/checks/AbstractFunctionNameCheck.java

```diff
@@ -36,10 +39,16 @@
   public String format = DEFAULT;
   private Pattern pattern = null;
 
+  private Pattern pattern() {
+    if (pattern == null) {
```

Comment:
@ivandalbosco Yes, because `init()` does not exist anymore. It was removed to avoid requiring that the methods of a PythonCheck are called in a specific order.